src/main/client/src/components/@pages/ProfilePage.tsx

```diff
@@ -0,0 +1,45 @@
+import { Page } from '../@layout/Page'
+import { Text, Flex, Heading, CircularProgress, CircularProgressLabel, Wrap, WrapItem, Center } from '@chakra-ui/react'
+
+const challenges = [
+  {
+    name: 'Bucketlist',
+    value: 40
+  },
+  {
+    name: 'Riddle',
+    value: 73
+  },
+  {
+    name: 'QR kód',
+    value: 17
+  }
+]
+
+type ProfilePageProps = {}
+
+export const ProfilePage: React.FC<ProfilePageProps> = (props) => {
+  return (
+    <Page {...props}>
+      <Heading>Profil</Heading>
+      <Text fontSize="3xl">Kovács Péter</Text>
+      <Text marginBottom="10" fontSize="3xl">
+        Tankör: I06
+      </Text>
+      <Wrap spacing="3rem" justify="center">
+        {challenges.map((challenge) => (
+          <WrapItem key={challenge.name}>
```

Comment:
Almost there, I think movig this to a a dedicated component after the ProfilePage component would be nicer. Then there is no need to create a challenges object and map them, because it will be an extra layer between the API call and the view. Just list them (since there is only three).
For now I approve this solution, just follow these steps when implementing the API :)